Kubernetes/helmcharts/azure-policy-addon-arc-connected-clusters/templates/deployment.yaml

```diff
@@ -1,12 +1,12 @@
-{{- if (and (ne .Values.azurepolicy.env.resourceid "<your_resource_id>") (and (ne .Values.azurepolicy.env.clientid "<your_client_id>") (ne .Values.azurepolicy.env.clientsecret "<your_client_secret>"))) }}
+{{- if (and (ne .Values.azurepolicy.env.resourceid "<your_resource_id>") (and (ne .Values.azurepolicy.env.tenantid "<your_tenant_id>") (and (ne .Values.azurepolicy.env.clientid "<your_client_id>") (ne .Values.azurepolicy.env.clientsecret "<your_client_secret>")))) }}
 apiVersion: v1
 kind: Secret
 metadata:
   name: azure-policy
   namespace: kube-system
 type: Opaque
 data:
-  client-secret: {{ .Values.azurepolicy.env.clientsecret | b64enc }}
+  client-secret: {{ .Values.azurepolicy.env.clientsecret | b64enc | quote }}
```

Comment:
Is quote needed? Can we remove it?

Reply:
It's needed. Ensures the secret gets deployed with quotation marks around it which ensures no ambiguous characters get picked up